tests/sharedtria/tria_zorder_01.mpirun=3.output

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
please rename this (and the other output files that need p4est) to contain ``.with_p4est=true.``